### PR TITLE
Fixes #13109: Add build id to quality assistant message

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/13109

### CODE REVIEW

File: scripts/rudder-dev/quality-assistant [hunk 1]
```diff
@@ -345,9 +349,9 @@ if __name__ == "__main__":
   Config.force = arguments['-f'] or arguments['--force']
   if arguments['merge'] and arguments['all']:
     list_repositories()
-    repo_merge_all(arguments['--alt-dir'])
+    repo_merge_all(arguments['--alt-dir'], arguments['--joib-id'])
```
Comment (amousset) [inline scripts/rudder-dev/quality-assistant]: job-id

File: scripts/rudder-dev/quality-assistant [hunk 2]
```diff
@@ -345,9 +349,9 @@ if __name__ == "__main__":
...
   elif arguments['merge']:
-    repo_merge(arguments['<repo>'], arguments['--alt-dir'])
+    repo_merge(arguments['<repo>'], arguments['--alt-dir'], arguments['--joib-id'])
```
Comment (amousset) [inline scripts/rudder-dev/quality-assistant]: job-id

**Comment by peckpeck** [comment]
Commit modified

File: scripts/rudder-dev/quality-assistant [hunk 3]
```diff
@@ -160,7 +160,7 @@ def repo_merge(repo, alt_dir):
     # Merge approved PR outside of freeze period
     elif not freeze:
       if Config.LOGLEVEL == "debug" or Config.LOGLEVEL == "info":
-        print("Merging approved PR: " + pr_info['html_url'])
+        print("Merging approved PR: " + pr_info['html_url'], jobid)
       pr_merge(repo, pr_info['html_url'], alt_dir)
```
Comment (amousset) [inline scripts/rudder-dev/quality-assistant]: jobid should be passed here

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**